FinalProject/client_secrets.json

```diff
@@ -1,9 +1 @@
-{
-  "web": {
-    "client_id": "PASTE_YOUR_CLIENT_ID_HERE",
-    "client_secret": "PASTE_YOUR_CLIENT_SECRET_HERE",
-    "redirect_uris": [],
-    "auth_uri": "https://accounts.google.com/o/oauth2/auth",
-    "token_uri": "https://accounts.google.com/o/oauth2/token"
-  }
-}
+{"web":{"auth_uri":"https://accounts.google.com/o/oauth2/auth","client_secret":"783UvsT9BeikibgfPmqCkjA7","token_uri":"https://accounts.google.com/o/oauth2/token","client_email":"454998953841-9n4nu563d9lvv242noc8o0m8np0ducrc@developer.gserviceaccount.com","redirect_uris":["https://www.example.com/oauth2callback"],"client_x509_cert_url":"https://www.googleapis.com/robot/v1/metadata/x509/454998953841-9n4nu563d9lvv242noc8o0m8np0ducrc@developer.gserviceaccount.com","client_id":"454998953841-9n4nu563d9lvv242noc8o0m8np0ducrc.apps.googleusercontent.com","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","javascript_origins":["https://www.example.com","http://localhost:5000","http://127.0.0.1:5000"]}}
```

Comment:
@jon-engelbert We don't need to provide our own credentials details. With above parameters the OAuth will still work.